app/javascript/groceries/components/check_in_items_list.vue

```diff
@@ -3,7 +3,7 @@ section(v-if="items.length > 0")
   h3.font-bold.mb-2 {{ title }} ({{ items.length }})
 
   ul.check-in-items-list.text-base.mb-2
-    li.flex.items-center.mb-2(
+    li.flex.items-center.break-word.mb-2(
```

Comment:
This makes sure that long words break (on word boundaries) rather than making the `li` indefinitely wide if the element contains length text.